cli/azd/pkg/azureutil/location.go

```diff
@@ -27,9 +26,10 @@ func (s Locs) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 
 // PromptLocation asks the user to select a location from a list of supported azure location
 func PromptLocation(
-	ctx context.Context, env *environment.Environment, message string, help string, console input.Console, azCli azcli.AzCli,
+	ctx context.Context, env *environment.Environment, message string, help string, console input.Console,
+	accountManager account.Manager,
 ) (string, error) {
-	return PromptLocationWithFilter(ctx, env, message, help, console, azCli, func(acl azcli.AzCliLocation) bool {
+	return PromptLocationWithFilter(ctx, env, message, help, console, accountManager, func(acl azcli.AzCliLocation) bool {
 		return true
 	})
```

Comment:
nit: Can we pass `nil` for the filter and have the internal usage set a default filter that returns true?

Reply:
Not going to address this for now given scope.